src/pages/workspaces/workspace/Data.js

```diff
@@ -732,15 +732,16 @@ export const WorkspaceData = _.flow(
       // setWdsProxyUrl({ status: 'Loading', state: '' })
```

Comment:
Was this intended to be used at some point?

Reply:
Not sure...it was a memento left here by my predecessor.  But since I didn't end up using it, I'll remove it.